kiwi/package_manager/zypper.py

```diff
@@ -149,11 +157,15 @@ def update(self):
         chroot_zypper_args = self.root_bind.move_to_root(
             self.zypper_args
         )
+        chroot_command_env = self.command_env
+        chroot_command_env['ZYPP_CONF'] = self.root_bind.move_path_to_root(
+            self.command_env['ZYPP_CONF']
+        )
```

Comment:
I think this is needed, this is to update ZYPP_CONF path to the chrooted env. It is true that the var is passed, but it is passed with a wrong path (i.e. `/tmp/image/build/image-root/tmp_config_file` should be converted to something like `/tmp_config_file`). It happens that exclude docs (and any other configuration in the conf file) is not applied in image packages, since zypper is executed in a chrooted operation and cannot find the conf file, so all defaults are applied.